src/server/pfs/server/driver.go

```diff
@@ -228,7 +228,7 @@ func (d *driver) inspectRepo(txnCtx *txncontext.TransactionContext, repo *pfs.Re
 	}
 	repoInfo := &pfs.RepoInfo{}
 	if err := d.repos.ReadWrite(txnCtx.SqlTx).Get(repo, repoInfo); err != nil {
-		return nil, errors.EnsureStack(err)
+		return nil, errors.EnsureStack(pfsserver.ErrRepoNotFound{Repo: repo})
```

Comment:
This makes me slightly uncomfortable, since we are losing the `err` from  `.Get(repo)`, but I'm making an assumption that `.Get(repo)` errors can be translated to `RepoNotFound`. An alternative to this approach could be to add an `error` field to `ErrRepoNotFound`, so that it wraps whatever error is returned by `.Get(repo)`.